### DISCUSSION TITLE
footprint on specific region

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1100] [Questions / Documentation] [answered]
Hello,

My understanding of footprinting is that it is an average of all the motifs across the _entire_ genome. Below, I have the default plot of a TF, which does seem to be bound to my "RNA-1" group across all motifs.
![Screen Shot 2021-10-22 at 12 40 06 PM](https://user-images.githubusercontent.com/92815743/138513600-47367262-60e0-45c0-a18f-a8e267718324.png)

However, this doesn't seem useful for answering the question of whether my TF is bound to the _particular_ cis-regulatory region of interest. I would like to zoom in to a specific window on the chromosome of a gene my TF interacts with. Here is what I have done to isolate the motifs in that locus:
```
ebf1 <- motifPositions["EBF1_67"]
q <- GRanges(seqnames = "chr22",
             ranges = IRanges(start = 45537600, end = 45538100),
             strand="+")

roi0 <- subsetByOverlaps(ebf1$EBF1_67, q)

roi1 <- GRangesList(EBF1_67 = spec)

seFoot <- getFootprints(
  ArchRProj = proj, 
  positions = roi1, 
  groupBy = "Clusters_rna"
)

plotFootprints(
  seFoot = seFoot,
  ArchRProj = proj,
  normMethod = "Subtract",
  plotName = "Footprints-Subtract-Bias",
  addDOC = FALSE,
  smoothWindow = 20,
  baseSize = 10
)
```

Which results in the following 
![Screen Shot 2021-10-22 at 12 41 23 PM](https://user-images.githubusercontent.com/92815743/138513744-5b8dcc2d-b8c5-48ca-b3c9-098d15d3427e.png)

I was wondering if you could help me understand a few things:
- Why is there more noise in the restricted region? I suppose I could use the `smoothWindow` arg in `plotFootprints` to tone this down, but I guess I expected noise relative to the genome-wide footprint.
- Should I be looking at the bottom plot instead? I don't really understand what "Tn5−Bias Normalized Insertions" means.
- Is this restricted footprint search something that is even recommended?

Thanks again for your time

**Comment by rcorces** [comment, marked as answer]
Footprinting at a single locus is essentially impossible given the depth of any standard ATAC-seq experiment. ArchR supports meta footprinting which I agree is mostly useless. But we dont have any support for single-site footpriting

**Reply by mitchTwoTimes** [reply]
Thanks so much for your response. This is 10X multiome data, so it has the depth of single nuc sequencing. Does that help my in my interpretation at all?

**Reply by rcorces** [reply]
unfrotunately, I just dont think this data has the depth to do single site footprinting (plus the Tn5 bias correction is still challenging). Maybe someone has done it but its extremely rare to see this done with ATAC-seq data in general (regardless of sc or bulk). I cant think of a reference that has done this.
